config/checkstyle/checkstyle.xml

```diff
@@ -82,9 +82,7 @@
 
         <!-- Checks for imports                              -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
-        <module name="AvoidStarImport">
-          <property name="allowStaticMemberImports" value="true"/>
-        </module>
+        <module name="AvoidStarImport"/>
```

Comment:
could you keep this?

Reply:
@fzrvic please revert this and changes in the previous files.

Reply:
reverted